sites/novascotiajobs/1-job-99.html

```diff
@@ -0,0 +1,12 @@
+---
+layout: default
+title: BC Jobs - 7 Day Free Trial
+landing-background-image: '/assets/img/handshake.jpg'
+permalink: novascotiajobs/7-day-trial-b
+post_url: "https://www.novascotiajobs.ca/coupons/7-day-trial-blue"
+region: Nova Scotia
+site_url: NovaScotiaJobs.ca
+site_title: Nova Scotia Jobs
+---
+
+{% include pages/7-day-trial-b.html %}
```

Comment:
Wrong reference?

Reply:
Thanks for catching. Too much copy & pasting.